README.rst

```diff
@@ -37,6 +37,10 @@ Install development requirements with::
 
     pip install -r requirements/local.txt
 
+Install additional dependencies::
+
+    apt-get install chromium-chromedriver
```

Comment:
Would be nice to add other platforms: OSX, Windows

https://sites.google.com/a/chromium.org/chromedriver/downloads